Model/Transaction.php

```diff
@@ -141,6 +141,31 @@ public function getTransactionOrder()
         return $oOrder;
     }
 
+    /**
+     * Returns transaction's payment method name.
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    public function getTransactionPaymentMethodName()
+    {
+        if ($this->getPaymentType()) { // there is an order associated with the transaction
```

Comment:
I would prefer having a **local variable** called avoiding redundancy by getting the payment type twice.

Reply:
👍